.changeset/orange-flowers-look.md

```diff
@@ -0,0 +1,7 @@
+---
+"@mds-core/mds-geography": minor
+"@mds-core/mds-policy": minor
```

Comment:
I think these should be major changes, as they are breaking.

Reply:
Looks good otherwise.

Reply:
Every package we have is on v0.x.y, and for semver any version <1.0.0 has breaking interface changes as minor releases instead of major. Making this a major bump makes it the first set of packages we have that's breaking outside of the <1.0.0 semver. IMO it'd probably be best to get everything onto major version semver all at once, so when generating changesets it's clear what bumping strategy should be used. 

My suggestion would be that this should be a `minor` bump for now.

Reply:
Ah, okay, I didn't know about this nuance of semver! Thanks for pointing it out.

Reply:
@marie-x change it back to minor I guess